### PR TITLE
Some additional updates to text_generation.ipynb

### PR DESCRIPTION
**Pull request by MarkDaoust**
* Re-save with colab.
* Fix top buttons
* Some variable name fixes
* Adjusted headings
* Use `stateful=True` to hide the RNN state.

### CODE REVIEW

File: site/en/tutorials/sequences/text_generation.ipynb [hunk 1]
```diff
@@ -728,14 +765,54 @@
     },
     {
       "metadata": {
-        "id": "tyvpYomYQQkF",
+        "id": "LyeYRiuVjodY",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "The model only accepts a fixd batch size. To use the same weights, and a different model we need to rebuild the model, and restore the wrights from the checkpoint.\n"
```
Comment (hereismari) [inline site/en/tutorials/sequences/text_generation.ipynb]: fixd->fixed

Comment (MarkDaoust) [reply site/en/tutorials/sequences/text_generation.ipynb]: Done. Thanks.

**Comment by random-forests** [comment]
Thank you! Typo: "restore the wrights"

**Comment by MarkDaoust** [comment]
Thanks Josh, this is fixed in the main PR now.

**Pull request merged**